packages/ember-toucan-core/src/components/form/file-input/delete-button.hbs

```diff
@@ -1,16 +1,15 @@
-<Button
-  @variant="bare"
-  type="button"
+<button
```

Comment:
I changed this to a `<button` and manually adjusted the styling as it was easier